packages/styles/settings-tools/_s.buttons.scss

```diff
@@ -25,10 +25,6 @@ $button-sizes: (
 );
 
 $icons: (
```

Comment:
This map and the modifiers are useless if the rule change. It's a risk to let users employ invalid parameters.

Reply:
fixed